app/design/adminhtml/default/default/template/sales/order/view/info.phtml

```diff
@@ -48,7 +48,7 @@ $emailSentInfo  = $_order->getEmailSent() ? Mage::helper('sales')->__('Email sen
                 <td class="value"><strong><span id="order_status"><?php echo $_order->getStatusLabel() ?></span></strong></td>
             </tr>
             <tr>
-                <td class="label"><label><?php echo Mage::helper('sales')->__('Order Confirmation') ?></label></td>
+                <td class="label"><label><?php echo Mage::helper('sales')->__('Order Confirmation Email') ?></label></td>
```

Comment:
I would keep "Order Confirmation" as it was initially, for several reasons. It is obvious that it is about sending an electronic message, the usage of the word Email in the phrase is not appropriate. The second reason is that in that block all the other labels are made up of two words, Order Confirmation should be no exception.

The change must be made in Mage_Sales.csv too.

Reply:
Thanks for suggestion, but please use https://egghead.io/lessons/github-add-suggestions-in-a-github-pr-review.

This would also save you time to comment.

Reply:
> I would keep "Order Confirmation" as it was initially, for several reasons. It is obvious that it is about sending an electronic message

Noit, its not.

I dont get it ... i'd like to fix your found bug, added good suggestion made by @justinbeaty and now we are talking about wording - again.

:angry: